AZ3166/src/cores/arduino/az_iot/deps/uhttp/src/uhttp.c

```diff
@@ -824,7 +820,7 @@ static int construct_http_headers(HTTP_HEADERS_HANDLE http_header, size_t conten
             else
             {
                 /* Codes_SRS_UHTTP_07_015: [on_bytes_received shall add the Content-Length http header item to the request.] */
-                if (sprintf(content, "%s: %lu%s", HTTP_CONTENT_LEN, content_len, HTTP_CRLF_VALUE) <= 0)
+                if (sprintf(content, "%s: %d%s", HTTP_CONTENT_LEN, (int)content_len, HTTP_CRLF_VALUE) <= 0)
```

Comment:
we have a big pr to sdk team which change all %zu to %lu because the ARM gcc not support %zu format specifier
here, why use %d and then do type convert?

Reply:
Yep, did that. Also updated the other `zu` as well.